example/single/single.cpp

```diff
@@ -0,0 +1,146 @@
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+#include <arbor/load_balance.hpp>
+#include <arbor/mc_cell.hpp>
+#include <arbor/morphology.hpp>
+#include <arbor/swcio.hpp>
+#include <arbor/simulation.hpp>
+#include <arbor/simple_sampler.hpp>
+
+#include <sup/tinyopt.hpp>
+
+struct options {
+    std::string swc_file;
+    double t_end = 20;
+    double dt = 0.025;
+    float syn_weight = 0.01;
+};
+
+options parse_options(int argc, char** argv);
+arb::morphology default_morphology();
+arb::morphology read_swc(const std::string& path);
+
+struct single_recipe: public arb::recipe {
+    explicit single_recipe(arb::morphology m): morpho(m) {}
+
+    arb::cell_size_type num_cells() const override { return 1; }
+    arb::cell_size_type num_probes(arb::cell_gid_type) const override { return 1; }
+    arb::cell_size_type num_targets(arb::cell_gid_type) const override { return 1; }
+
+    arb::probe_info get_probe(arb::cell_member_type probe_id) const override {
+        arb::segment_location mid_soma = {0, 0.5};
+        arb::cell_probe_address probe = {mid_soma, arb::cell_probe_address::membrane_voltage};
+
+        return {probe_id, 0, probe};
+    }
+
+    arb::cell_kind get_cell_kind(arb::cell_gid_type) const override {
+        return arb::cell_kind::cable1d_neuron;
+    }
+
+    arb::util::unique_any get_cell_description(arb::cell_gid_type) const override {
+        arb::mc_cell c = make_mc_cell(morpho);
+
```

Comment:
Because this is example code, I think it is worth a little comment along the lines of
```
// Add hh mechanism to the soma, and have passive dendrites.

Reply:
I'll add a comment along the lines of the description int he readme.